circle.yml

```diff
@@ -85,6 +85,23 @@ jobs:
     steps:
       - attach
       - expect
+      - add_ssh_keys:
+          fingerprints:
+            - '16:04:2d:ed:15:8d:0b:56:ea:ee:82:6e:01:a4:5b:09'
+      - run:
+          name: Add Github Key
+          command: ssh-keyscan -H github.com >> ~/.ssh/known_hosts
+      - run:
+          name: Configure Git Config & Branch
+          command: |
+            git status
+            git config --global user.email "tds.github.bot@gmail.com"
+            git config --global user.name "TDSBot"
+      - run:
+          name: Rebase branch
+          command: |
+            git checkout master
```

Comment:
Can we just `git rebase master`? Wouldn't we have a clean master that we can rebase from?

Reply:
Unfortunately no. It seems that circle ci actually adds commits from that branch to the top of its local master on its own, so rebasing won't work. Kind of odd behaviour but I can't find anything we did that does that so it may just be a normal thing.